commons/src/main/kotlin/androidx/lifecycle/LiveDataHiddenApi.kt

```diff
@@ -0,0 +1,7 @@
+package androidx.lifecycle
```

Comment:
Add in references where we've got this from. Make updates as needed by codacy too.